requirements.txt

```diff
@@ -6,7 +6,7 @@ chardet==3.0.4
 coverage==4.5.1
 # for build --parallel
 -e git://github.com/docker/compose@473703d0d9e0f81520dd41aa30da67d7868f877a#egg=docker-compose
-docker-compose-wait==1.0.0
+docker-compose-wait==1.0.2
```

Comment:
let's be sure to give the team a heads up, if this is required

Reply:
I have faced this issue locally https://github.com/nicolas-van/docker-compose-wait/issues/10 because of that I push it to 1.0.2